setup.py

```diff
@@ -71,9 +73,11 @@ def run(self):
         'git://github.com/NeuralEnsemble/libNeuroML.git#egg=libNeuroML',
         'git://github.com/zopefoundation/ZODB.git#egg=ZODB',
         'git://github.com/RDFLib/FuXi#egg=FuXi-1.4.dev-r0',
+        'git+https://github.com/RDFLib/rdflib-zodb#egg=rdflib-zodb-1.1-dev',
+        'git+https://github.com/mwatts15/YAROM@dev#egg=yarom-0.7.1',
```

Comment:
Can you release YAROM to PyPI first if we are going to incorporate it into PyOpenWorm?  The more we add dependencies on unreleased code, the more brittle PyOpenWorm becomes.